docs/en/stack/ml/anomaly-detection/ootb-ml-jobs.asciidoc

```diff
@@ -0,0 +1,931 @@
+
+* For APM data where `agent.name` is `js-base`.
+* Models the request rate of user agents (`partition_field_name` is 
+  `user_agent.name`).
+* Detects user agents that are making requests at a suspiciously high rate 
+  (using the <<ml-nonzero-count,`high_non_zero_count` function>>).
+
+This job is useful in identifying bots.
+
+////
+Influencers:
+
+* `service.name`
+* `user_agent.name`
+
+Bucket span: 15m.
+
+Function: `high_non_zero_count`.
+////
+
+high_mean_response_time::
+
+* For transaction data where `processor.event` is `transaction` and 
+`transaction.type` is `request`.
+* Models response time duration of transactions.
+* Detects anomalies in high mean of transaction duration (using the 
+  <<ml-metric-mean,`high_mean` function>>).
+
+////
+Bucket span: 15m.
+
+Function: `high_mean`.
+////
+// end::apm-jobs[]
+
+[float]
+[[ootb-ml-jobs-auditbeat]]
+=== Auditbeat
+
+// tag::auditbeat-jobs[]
+These {anomaly-job} wizards appear in {kib} if you use 
+{auditbeat-ref}/index.html[{auditbeat}] to audit process activity on your 
+systems.
+
+docker_high_count_process_events_ecs::
+
+* For Auditbeat data where `event.module` is `auditd` and `container.runtime` is 
+`docker`.
+* Models process execution rates (`partition_field_name` is `container.name`).
+* Detects unusual increases in process execution rates in Docker containers 
+  (using the <<ml-count,`high_count` function>>).
+
+////
+Influencers:
+
+* `container.name`
+* `process.executable`
+
+Bucket span: 1h.
+
+Function: `high_count`.
+////
+
+docker_rare_process_activity_ecs::
+
+* For Auditbeat data where `event.module` is `auditd` and `container.runtime` is 
+`docker`.
+* Models occurrences of process execution (`partition_field_name` is 
+  `container.name`).
+* Detects rare process executions in Docker containers (using the 
+  <<ml-rare,`rare` function>>).
+
+////
+Influencers:
+
+* `container.name`
+* `process.executable`
+
+Bucket span: 1h.
+
+Function: `rare`.
+////
+
+hosts_high_count_process_events_ecs::
+
+* For Auditbeat data where `event.module` is `auditd`.
+* Models process execution rates (`partition_field_name` is `host.name`).
+* Detects unusual increases in process execution rates (using the 
+  <<ml-nonzero-count,`high_non_zero_count` function>>).
+
+////
+Influencers:
+
+* `host.name`
+* `process.executable`
+
+Bucket span: 1h.
+
+Function: `high_non_zero_count`.
+////
+
+hosts_rare_process_activity_ecs::
+
+* For Auditbeat data where `event.module` is `auditd`.
+* Models process execution rates (`partition_field_name` is `host.name`).
+* Detects rare process executions on hosts (using the 
+  <<ml-rare,`rare` function>>).
+
+////
+Influencers:
+
+* `host.name`
+* `process.executable`
+
+Bucket span: 1h.
+
+Function: `rare`.
+////
+// end::auditbeat-jobs[]
+
+[float]
+[[ootb-ml-jobs-logs-ui]]
+=== Logs UI
+
+// tag::logs-jobs[]
+These {anomaly-jobs} appear by default in the {kibana-ref}/xpack-logs.html[Logs app] in {kib}. 
+log_entry_categories_count::
```

Comment:
This job name isn't formatted properly in the preview. I think the solution is to have a preceding blank line:
```suggestion

log_entry_categories_count::
```